Sources/GeneratorCLI/GeneratorCLI.swift

```diff
@@ -40,18 +39,7 @@ struct GeneratorCLI: AsyncParsableCommand {
         logger: logger
       )
 
-      let serviceGroup = ServiceGroup(
-        configuration: .init(
-          services: [.init(
-            service: SwiftSDKGeneratorService(recipe: recipe, generator: generator),
-            successTerminationBehavior: .gracefullyShutdownGroup
-          )],
-          cancellationSignals: [.sigint],
```

Comment:
We need to reimplement functionality introduced by `cancellationSignals: [.sigint]` if this is to be removed, otherwise Ctrl-C in terminal on a running generator process doesn't send signals to child processes. I assume corresponding code in `swift-service-lifecycle` is self-contained and we can just vendor it here?